### ISSUE TITLE
Parameter random_state not working as expected

### ISSUE THREAD

**Issue by Yatoom** [rhiever/tpot#329]
I noticed that locking the random seed to a fixed number did not seem to have effect, as each new run 
came up with a different pipeline and different parameters. @rhiever asked me to file a bug report, and added that this feature might have broken at some point.

## Context of the issue

Passing the `random_state` parameter to TPOTClassifier or TPOTRegressor does not have the desired effect of reproducibility, but generates different pipelines on each execution.

## Process to reproduce the issue
Here is a minimal coding example to reproduce the issue. Running this script multiple times will result in different pipelines.
```python
from sklearn.datasets import load_digits
data = load_digits()
X = data.data
y = data.target

from tpot import TPOTClassifier
tpot = TPOTClassifier(generations=1, population_size=1, verbosity=2, random_state=1)
tpot.fit(X, y)
```
